docs/usage/CATEGORY_SYNC.md

```diff
@@ -138,4 +142,25 @@ More examples of those utils for different fields can be found [here](/src/integ
 ## Caveats
 
 1. Categories are either created or updated. Currently the tool does not support category deletion.
-2. The library will sync all field types of custom fields, except `ReferenceType`. [#87](https://github.com/commercetools/commercetools-sync-java/issues/3). 
+2. The sync library is not meant to be executed in a parallel fashion. For example:
+    ````java
+    final CategorySync categorySync = new CategorySync(syncOptions);
+    final CompletableFuture<CategorySyncStatistics> syncFuture1 = categorySync.sync(batch1).toCompletableFuture();
+    final CompletableFuture<CategorySyncStatistics> syncFuture2 = categorySync.sync(batch2).toCompletableFuture();
+    CompletableFuture.allOf(syncFuture1, syncFuture2).join;
+    ````
+    The aforementioned example, demonstrates how the library should **not** be used. The library, however, should be instead
```

Comment:
example without comma would be nicer for not separating the subject and verb.

Reply:
77a939a